src/controllers/hid/hidenumerator.cpp

```diff
@@ -120,10 +118,13 @@ QList<Controller*> HidEnumerator::queryDevices() {
             continue;
         }
 
-        HidController* newDevice = new HidController(std::move(deviceInfo));
-        m_devices.push_back(newDevice);
+        m_devices.push_back(std::make_unique<HidController>(std::move(deviceInfo)));
```

Comment:
not that independent of my changes, this is currently broken when querying devices multiple times because we don't clear the list from the previous query... Not sure if I should fix that here because its not feasible to test nor do I have enough non-keyboard/-mice HID devices here.

Reply:
I think we should fix it anyway. Main is still in alpha stage.